lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -302,10 +302,10 @@ def split_templates(filename)
       raise FatalDeploymentError, "Failed to render and parse template"
     end
 
-    def record_invalid_template(err:, filename:, content:)
+    def record_invalid_template(err:, filename:, content: nil)
       debug_msg = ColorizedString.new("Invalid template: #{filename}\n").red
       debug_msg += "> Error message:\n#{FormattedLogger.indent_four(err)}"
```

Comment:
shouldn't we also be suppressing (or replacing) the error itself? Just because it had a filename in it doesn't really tell us what else it contains

Reply:
Like this? https://github.com/Shopify/kubernetes-deploy/pull/438/commits/97a4fd30896fc86ab8545b80b0b33079754c126e